# FootOutputMsg.msg

time stamp
int16 id

float32 x
float32 y
float32 phi
float32 theta
float32 psi

float32 Fx
float32 Fy
float32 Tphi
float32 Ttheta
float32 Tpsi

float32 vx
float32 vy
float32 wphi
float32 wtheta
float32 wpsi

int16 state